ark-testing/src/context.rs

```diff
@@ -1,8 +1,7 @@
 use std::path::PathBuf;
-use std::fs;
+use std::fs::{remove_dir_all, create_dir_all};
```

Comment:
nit: Personally, I actually like to use the stdlib `fs::Xxx` prefixes, same with `io::` etc. They're short and you know right away from reading the code that it's a stdlib function so no side effects.